docs/user/tutorials/01-30-set-up-mtls-gateway.md

```diff
@@ -0,0 +1,134 @@
+# Expose workloads behind mTLS-enabled Gateway
+
+This document showcases how to set up a mTLS Gateway in Istio and expose it with APIRule.
+
+According to the official [CloudFlare documentation](https://www.cloudflare.com/learning/access-management/what-is-mutual-tls/):
+>Mutual TLS, or mTLS for short, is a method for mutual authentication. mTLS ensures that the parties at each end of a network connection are who they claim to be by verifying that they both have the correct private key. The information within their respective TLS certificates provides additional verification.
+
+To have a working mTLS connection, several things are required:
+
+1. Working DNS entry pointing to the Istio Gateway IP
+2. Valid Root CA certificate and key
+3. Generated client and server certificates with a private key
+4. Istio and API-Gateway installed on a Kubernetes cluster
+
+The procedure of getting working mTLS Gateway are described in the following steps. For the needs of this tutorial, a Gardener shoot cluster and its API will be used.
+
+The mTLS Gateway is exposed under `*.mtls.example.com` with valid DNS `A` record.
+
+## Create DNS Entry and generate wildcard certificate
+
+> Note: This step is heavily dependent on the configuration of a hyperscaler. Always consult official documentation of each cloud service.
+
+For Gardener shoot clusters follow ["set up a custom domain for workload" guide](01-10-setup-custom-domain-for-workload.md).
+
+## Generate self-signed Root CA and client certificate
```

Comment:
We also have the tutorial [Prepare Self-Signed Root Certificate Authority and Client Certificates](https://kyma-project.io/#/api-gateway/user/tutorials/01-60-security/01-61-mtls-selfsign-client-certicate?id=prepare-self-signed-root-certificate-authority-and-client-certificates). If it's up to date, we could link it in prerequisites instead of adding this step

Reply:
Looks like it's the same thing and could be reused